lib/atscppapi/examples/helloworld/HelloWorldPlugin.cc

```diff
@@ -20,16 +20,23 @@
 #include <iostream>
 #include <atscppapi/GlobalPlugin.h>
 #include <atscppapi/PluginInit.h>
-
+#include <../../../../proxy/api/ts/ts.h>
```

Comment:
I think the plugin shouldn't include ts.h at all - it should call a wrapper function provided by the CPP API core, if for no other reason than avoiding this issue. Then we can at least isolate the ugliness in one spot and in the core.